Dockerfile.coverage

```
@@ -1,12 +1,14 @@
 # Dockerfile for test coverage generation
 # Based on main Dockerfile with Xdebug added
 
-FROM debian:bullseye
+FROM debian:trixie
 LABEL maintainer Linagora Folks <openpaas@linagora.com>
 
-ENV PHPVERSION=7.4
+ENV PHPVERSION=8.4
 
 ADD ./docker/config/20-apcu.ini /etc/php/${PHPVERSION}/fpm/conf.d/20-apcu.ini
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)